satellite/satellitedb/overlaycache.go

```diff
@@ -536,7 +534,7 @@ func (cache *overlaycache) Paginate(ctx context.Context, offset int64, limit int
 }
 
 // Update updates node address
-func (cache *overlaycache) UpdateAddress(ctx context.Context, info *pb.Node) (err error) {
+func (cache *overlaycache) UpdateAddress(ctx context.Context, info *pb.Node, defaults overlay.NodeSelectionConfig) (err error) {
```

Comment:
https://github.com/storj/storj/blob/b638d2560af0ca98d004443c4fb85e5adcec2e7e/satellite/satellitedb/overlaycache.go#L556

should there be any distinction be made between "no sql rows" vs any db get err?


similarly other places when Get_Node_By_Id() is called
dbNode, err := tx.Get_Node_By_Id(ctx, dbx.Node_Id(nodeID.Bytes()))
	if err != nil {
		return nil, Error.Wrap(errs.Combine(err, tx.Rollback()))
	}
like that. the err could be no sql rows and could abort the operation. Just an observation.